bokehjs/src/lib/models/plots/plot_canvas.ts

```diff
@@ -648,6 +678,19 @@ export class PlotView extends LayoutDOMView {
     }
   }
 
+  protected _paint_layout(ctx: Context2d, layout: Layoutable) {
```

Comment:
Would be nice to turn on wireframing from Python somehow

Reply:
I intend to do that at some point. Right now all this is more of a hack, because e.g. you have to trigger painting on your own for this to take effect. More to come in labeling work, because there you can also outline individual labels of an axis, etc.